outputparser/regex_dict.go

```diff
@@ -0,0 +1,87 @@
+package outputparser
+
+import (
+	"fmt"
+	"regexp"
+
+	"github.com/tmc/langchaingo/schema"
+)
+
+// RegexDict is an output parser used to parse the output of an llm as a map.
+type RegexDict struct {
+	OutputKeyToFormat map[string]string
+	NoUpdateValue     string
```

Comment:
I think both these fields would benefit from comments.

Reply:
Agreed. I had TODOs here to improve the naming / add commenting, but apparently I nixed that.